install-ast/action.yml

```
@@ -15,12 +15,12 @@ runs:
     - name: Set up dotnet
       uses: actions/setup-dotnet@2216f56ae1eec353f06abd464e2ec435fa5f5d43
```

```suggestion
      uses: actions/setup-dotnet@3447fd6a9f9e57506b15f895c5b76d3b197dc7c2 # v3.2.0  
```
Since we are here, we could update this.